app/services/create_interview.rb

```diff
@@ -0,0 +1,37 @@
+class CreateInterview
+  attr_reader :auth, :application_choice, :provider, :date_and_time, :location, :additional_details
+
+  def initialize(
+    actor:,
+    application_choice:,
+    provider:,
+    date_and_time:,
+    location:,
+    additional_details:
+  )
+    @auth = ProviderAuthorisation.new(actor: actor)
+    @application_choice = application_choice
+    @provider = provider
+    @date_and_time = date_and_time
+    @location = location
+    @additional_details = additional_details
+  end
+
+  def call
```

Comment:
```suggestion
  def save
```

We tend to used `save` or `save!` for the services that change the state e.g. AcceptOffer WithdrawApplication DeclineOffer MakeAnOffer etc